lib/gds_api/account_api.rb

```diff
@@ -43,6 +43,13 @@ def get_user(govuk_account_session:)
     get_json("#{endpoint}/api/user", auth_headers(govuk_account_session))
   end
 
+  # Delete a users account
+  #
+  # @param [String] govuk_account_session Value of the session header
+  def delete_user(govuk_account_session:)
+    delete_json("#{endpoint}/api/user", auth_headers(govuk_account_session))
+  end
```

Comment:
Like `update_user_by_subject_identifier`, this should use `"#{endpoint}/api/oidc-users/#{subject_identifier}"` - maybe call it `delete_user_by_subject_identifier` to match?

Reply:
In https://github.com/alphagov/account-api/pull/157 I've changed the code example to `delete_user_by_subject_identifier`